src/NzbDrone.Core/Configuration/ConfigFileProvider.cs

```diff
@@ -195,7 +195,7 @@ public string UrlBase
                     return urlBase;
                 }
 
-                return "/" + urlBase.Trim('/').ToLower();
+                return "/" + urlBase;
```

Comment:
Leave the `.Trim()`

Reply:
If you notice on line [191](https://github.com/Sonarr/Sonarr/pull/4820/files#diff-bb1f86bcc03982676d45aa15a713f388209474ee00645e920cc358780566b454R191) `urlBase` is already trimmed thus it is redundant to `Trim` the string again.

Reply:
`Trim('/')` trims leading and trailing `/`s in that case, `Trim()` here trims leading and trailing spaces, though changing `Trim('/')` to `Trim('/', ' ')` would prevent unexpected URL bases with just slashes and spaces from being set.

Reply:
I get what you're saying but like I said it's superfluous because it's already being applied when `urlBase` is set. Thus, if I were to re-add it:
```c#
var urlBase = GetValue("UrlBase", "").Trim('/');

if (urlBase.IsNullOrWhiteSpace())
{
    return urlBase;
}

return "/" + urlBase.Trim('/');
```

Would be equivalent to (ignoring the if statement):
```c#
return "/" + GetValue("UrlBase", "").Trim('/').Trim('/');
```

Reply:
Doh, kept reading the removed one as trim without arguments, which would legitimately remove extra spaces. Nothing to see here.